src/EA.Weee.RequestHandlers/Scheme/MemberRegistration/ProducerChargeCalculator.cs

```diff
@@ -7,34 +7,34 @@
     public class ProducerChargeCalculator : IProducerChargeCalculator
     {
         private readonly IProducerChargeCalculatorDataAccess dataAccess;
-        private readonly IProducerChargeBandCalculator producerChargeBandCalculator;
-
-        public ProducerChargeCalculator(IProducerChargeCalculatorDataAccess dataAccess, IProducerChargeBandCalculator producerChargeBandCalculator)
+        private readonly IProducerChargeBandCalculatorChooser producerChargeBandCalculatorChooser;
+        public ProducerChargeCalculator(IProducerChargeCalculatorDataAccess dataAccess, 
+            IProducerChargeBandCalculator producerChargeBandCalculator, 
+            IProducerChargeBandCalculatorChooser producerChargeBandCalculatorChooser)
         {
             this.dataAccess = dataAccess;
-            this.producerChargeBandCalculator = producerChargeBandCalculator;
+            this.producerChargeBandCalculatorChooser = producerChargeBandCalculatorChooser;
         }
 
-        public ProducerCharge CalculateCharge(string schemeApprovalNumber, producerType producer, int complianceYear)
+        public ProducerCharge CalculateCharge(schemeType scheme, producerType producer, int complianceYear)
         {
-            ProducerCharge producerCharge = GetProducerCharge(producer);
+            ProducerCharge producerCharge = GetProducerCharge(scheme, producer, complianceYear);
 
             if (producer.status == statusType.A)
```

Comment:
Just looking at the acceptance criteria 
7.Remove / comment out the code that deals with changes to charge bands (we'll update this in a separate story)
We should comment out this checking for A status... I think there will be a new rule coming in

Reply:
commented out